### DISCUSSION TITLE
bin/console error 139

### DISCUSSION THREAD

**Discussion by GeekComics** [symfony/symfony#45604] [Q&A] [answered]
### Symfony version(s) affected

6.1

### Description

I use an attribute route for controller
when i start bin/console cache:clear
i get error:

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 139
!!  
!!   // Clearing the cache for the dev environment with debug                       
!!   // true                                                                        
!!  
!!  Segmentation fault
!!  
Script @auto-scripts was called via post-install-cmd
```

If i change attribute for annotation it work

### How to reproduce

Add route attribute to controller

### Possible Solution

Change for annotation

**Comment by lyrixx** [comment]
Hello, there is a segfault, this means PHP crash brutally.

If can be caused by different options, like a bug in PHP (few probable), an infinite loop, or something else.

You can try to install xdebug, it may reveal the real issue.

**Comment by chyshyya** [comment]
Do you use php 8.1?

**Reply by OleksiiBulba** [reply]
I have a similar issue: when I install security bundle, I get the same error (`Segmentation fault (core dumped)`).
I used php 8.1.3, after your message I decided to update my php container to 8.1.7, after it builds, I will update with result if the issue is gone.

**Reply by OleksiiBulba** [reply]
Hey-ya, upgrading php 8.13 -> 8.1.7 solved the issue!

**Comment by OleksiiBulba** [comment, marked as answer]
Hello @GeekComics, I had similar issue (with the same error message) but in different condition (when I install symfony security bundle, cache:clear did not finish with the same as you error).
After I upgraded php version from 8.1.3 to 8.1.7 the issue is fixed, so possibly try to upgrade too, if you use php 8.
